### DISCUSSION TITLE
"Option A" in module configuration

### DISCUSSION THREAD

**Discussion by nonix** [arendst/Tasmota#11250] [Q&A] [answered]
Hi, anybody knows what the "Option A" in module configuration is used for?
I have std. tasmota (dev) compiled with additional add-ons: USE_Display + USE_Thermostat + USE_EXPRESSION;
![image](https://user-images.githubusercontent.com/1888777/110235580-18bb8680-7f31-11eb-8d43-f3f4aaa18e5f.png)

Thx a lot,
N.

**Comment by arendst** [comment, marked as answer]
Option_A1 - (Light) Change PWM1 to input on power off and no fade running (1)

**Reply by AcoranTf** [reply]
Could you please create a section where you can explain how all the configuration functions of the tasmota module work?

**Reply by sfromis** [reply]
Most pins have a short text in https://tasmota.github.io/docs/Components/ and are generally reasonably understandable when you know what's relevant to your hardware. The Option_A ones are of course not so easy to guess; here's the current available set:
``` 
Option_A1 (Light) Change PWM1 to input on power off and no fade running (1)
Option_A2 (Energy) Enable dummy values
Option_A3 (Display) Universal display driver
Option_A4 (Zigbee) Enable CCLoader using Zigbee Rx/Tx/Rst Gpios
Option_A5 (Rotary) Enable Mi Desk emulation
Option_A6 (Light) LinkInd support
Option_A7 (Device) Shelly Pro
```

**Reply by AcoranTf** [reply]
Thanks, but I was referring to all the options. I've already looked at the Components section, but there's a lot that isn't clear at all.
